### PR TITLE
core: handle TODO introduced while refactoring infra load

### PR DESCRIPTION
**Pull request by bougue-pe**
Mainly on behavior introduced only to match previous load (and ease comparison).
As for the TODO on SpeedLimit, an issue has been created: https://github.com/OpenRailAssociation/osrd/issues/7794.

Fixes part of #7056

### CODE REVIEW

**Review by Khoyo — APPROVED** [review]

File: core/kt-osrd-rjs-parser/src/main/kotlin/fr/sncf/osrd/RawInfraRJSParser.kt [hunk 1]
```diff
@@ -801,7 +793,10 @@ fun parseRJSInfra(rjsInfra: RJSInfra): RawInfra {
                     props
                 )
             if (partId == null) {
-                // TODO warning
+                // TODO: raise warning
```
Comment (Castavo) [inline core/kt-osrd-rjs-parser/src/main/kotlin/fr/sncf/osrd/RawInfraRJSParser.kt]: Isn't the warning raised now ?

**Review by Castavo — APPROVED** [review]
LGTM, but frankly I can't say I understand very well what I read.

File: core/kt-osrd-rjs-parser/src/main/kotlin/fr/sncf/osrd/RawInfraRJSParser.kt (hunk 1 above)
Comment (bougue-pe) [reply core/kt-osrd-rjs-parser/src/main/kotlin/fr/sncf/osrd/RawInfraRJSParser.kt]: To my understanding, the point of @eckter here is to add a warning to the response returned to editoast (or at least link the trace to a span of editoast) as mentionned here https://github.com/OpenRailAssociation/osrd/issues/6925.
I only did the easy part here, which I think is useful but not enough to cover the TODO.

Maybe I should clarify the todo: `link warning to specific request (through response or tracing)` ?

Comment (Castavo) [reply core/kt-osrd-rjs-parser/src/main/kotlin/fr/sncf/osrd/RawInfraRJSParser.kt]: Yeah, specifying would be perfect

Comment (bougue-pe) [reply core/kt-osrd-rjs-parser/src/main/kotlin/fr/sncf/osrd/RawInfraRJSParser.kt]: Done (amended) in https://github.com/OpenRailAssociation/osrd/pull/7792/commits/7d8c9e80d458828f1e0c23bd2404e56b070e4af3#diff-9325fe5faad772881e72d6024d73bade0606de7420d369192dd9486480914c48R796

**Pull request merged**